python/ray/data/read_api.py

```diff
@@ -1797,12 +1824,22 @@ def convert(ds: "datasets.Dataset") -> Dataset:
         return ray_ds
 
     if isinstance(dataset, datasets.DatasetDict):
+        available_keys = list(dataset.keys())
+        logger.warning(
```

Comment:
this is great! I wonder if this explicit logic should also go into the docstring of `from_hugginface` directly (as a follow-up)